src/egon/data/datasets/pypsaeur/__init__.py

```diff
@@ -1065,11 +1065,21 @@ def links_to_etrago(neighbor_links, scn="eGon100RE", extendable=True):
             index_label="link_id",
         )
 
-    non_extendable_links_carriers = [
-        "H2 pipeline retrofitted",
-        "H2 pipeline",
-        "gas pipeline",
-        "biogas to gas",
+    extendable_links_carriers = [
+        "battery charger",
+        "battery discharger",
+        "home battery charger",
+        "home battery discharger",
+        "rural water tanks charger",
+        "rural water tanks discharger",
+        "urban central water tanks charger",
+        "urban central water tanks discharger",
+        "urban decentral water tanks charger",
+        "urban decentral water tanks discharger",
+        "H2 Electrolysis",
+        "H2 Fuel Cell",
+        "SMR",
+        "Sabatier",
```

Comment:
Looks good to me. 
Some of the carrier names are still not in line with the ones we use within Germany. But that is maybe another issue, right?

Reply:
at this point, it should or even has to work with the p-e wording since it is at that point still not changed. You are right it is another issue but I think most of the names are tackled already. But I will check if we can add s.th. there.